.github/workflows/ci.yml

```diff
@@ -0,0 +1,77 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - development
+      - releases/*
```

Comment:
I don't think we need to explicitly build the `releases/*` branches, I added this while testing...

```suggestion
```

Reply:
Yeah, that makes sense to me